library/src/main/java/com/woxthebox/draglistview/ColumnProperties.java

```diff
@@ -0,0 +1,218 @@
+package com.woxthebox.draglistview;
```

Comment:
Add Apache License in top as in other files. Also add in ViewUtils file

Reply:
Added